pkg/ai/mcp/backplane_console.go

```diff
@@ -39,34 +45,100 @@ func BackplaneConsole(ctx context.Context, request *mcp.CallToolRequest, input B
 				Content: []mcp.Content{
 					&mcp.TextContent{Text: fmt.Sprintf("Error setting browser flag: %v", err)},
 				},
-			}, struct{}{}, nil
+			}, nil, nil
 		}
 	}
 
-	// Call the console command's RunE function directly
-	err := consoleCmd.RunE(consoleCmd, args)
+	// Capture stdout to get the console URL
+	oldStdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
 
-	if err != nil {
-		errorMessage := fmt.Sprintf("Failed to access console for cluster '%s'. Error: %v", clusterID, err)
+	// Channel to capture the command execution result
+	resultChan := make(chan error, 1)
 
-		return &mcp.CallToolResult{
-			Content: []mcp.Content{
-				&mcp.TextContent{Text: errorMessage},
-			},
-		}, struct{}{}, nil // Return nil error since we're handling it gracefully
+	// Run the console command in a goroutine
+	go func() {
+		resultChan <- consoleCmd.RunE(consoleCmd, args)
+	}()
+
+	// Capture output with timeout to get the console URL
+	outputChan := make(chan string, 1)
+	go func() {
+		var buf bytes.Buffer
+		scanner := bufio.NewScanner(r)
+		for scanner.Scan() {
+			line := scanner.Text()
+			buf.WriteString(line + "\n")
+			// Once we see the console URL, we have what we need
```

Comment:
I'm not sure, is git a good idea to scan the output message and handle the MCP success message 
My thinking is that we can improve the console command to support the MCP endpoint rather than handling it here